deploy/run_with_summon.sh

```diff
@@ -5,39 +5,44 @@ set -xeuo pipefail
 
 # Clean up when script completes and fails
 function finish {
-  announce 'Wrapping up and removing test environment'
```

Comment:
this is actually good, just needs re-wording

Reply:
This should only happen when not in docker or dev I moved this time down